sdk/purview/purview-datamap-rest/src/outputModels.ts

```diff
@@ -594,17 +547,17 @@ export interface QueryResultOutput {
    * The total number of search results (not the number of documents in a single
    * page).
    */
-  searchCount?: number;
+  "@search.count"?: number;
```

Comment:
could you check the swagger or typespec for this?

Reply:
this change is because the previous was generated before we supported encoded name in codegen
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/purview/Azure.Analytics.Purview.DataMap/models.tsp#L1011